### PR TITLE
rating calculation and show

### PR DESCRIPTION
**Pull request by sujoydatta**

### CODE REVIEW

**Comment by MuhammadTamzid** [comment]
Ok

**Comment by zenith19** [comment]
ok

File: app/models/book.rb [hunk 1]
```diff
@@ -39,4 +41,8 @@ def ransackable_scopes auth_object = nil
       [:filter_book]
     end
   end
+
+  def init_rating
```
Comment (dieunb) [inline app/models/book.rb]: private

Comment (sujoydatta) [inline app/models/book.rb]: its in private

**Pull request merged**